### PR TITLE
CrossContainerExtension.php cleanup

### PR DESCRIPTION
**Pull request by Kibsoon**
Save the lines!

### CODE REVIEW

**Review by pamil — APPROVED** [review]

File: src/ServiceContainer/CrossContainerExtension.php [hunk 1]
```diff
@@ -61,7 +60,6 @@ public function initialize(ExtensionManager $extensionManager)
      */
     public function configure(ArrayNodeDefinition $builder)
     {
-
```
Comment (pamil) [inline src/ServiceContainer/CrossContainerExtension.php:64]: Good job! :+1:

**Review by lchrusciel — APPROVED** [review]
MUST - MERGE!

File: src/ServiceContainer/CrossContainerExtension.php [hunk 2]
```diff
@@ -51,7 +51,6 @@ public function getConfigKey()
      */
     public function initialize(ExtensionManager $extensionManager)
     {
-
```
Comment (pamil) [inline src/ServiceContainer/CrossContainerExtension.php:54]: Very interesting change!

File: src/ServiceContainer/CrossContainerExtension.php [hunk 3]
```diff
@@ -81,6 +79,5 @@ public function load(ContainerBuilder $container, array $config)
      */
     public function process(ContainerBuilder $container)
     {
-
```
Comment (pamil) [inline src/ServiceContainer/CrossContainerExtension.php:84]: Wouldn't even think such optimisation is possible.

**Review by pamil — APPROVED** [review]

**Pull request merged**